packages/contracts/contracts/core/IDAO.sol

```diff
@@ -13,121 +15,114 @@ abstract contract IDAO {
         bytes data; // FuncSig + arguments
     }
 
-    /// @notice Checks if an address has permission on a contract via a role identifier and considers if `ANY_ADDRESS` was used in the granting process.
-    /// @param _where The address of the contract
-    /// @param _who The address of a EOA or contract to give the permissions
-    /// @param _role The hash of the role identifier
-    /// @param _data The optional data passed to the ACLOracle registered
-    /// @return bool Returns whether the address has permission is or not
-    function hasPermission(
+    /// @notice Checks if an address has permission on a contract via a permissionID identifier and considers if `ANY_ADDRESS` was used in the granting process.
+    /// @param _where The address of the contract.
+    /// @param _who The address of a EOA or contract to give the permissions.
+    /// @param _permissionID The permission identifier.
+    /// @param _data The optional data passed to the `PermissionOracle` registered.
+    /// @return bool Returns true if the address has permission, false if not.
+    function checkPermission(
         address _where,
         address _who,
-        bytes32 _role,
+        bytes32 _permissionID,
         bytes memory _data
     ) external virtual returns (bool);
 
-    /// @notice Updates the DAO metadata
-    /// @dev Sets a new IPFS hash
-    /// @param _metadata The IPFS hash of the new metadata object
+    /// @notice Updates the DAO metadata (e.g., an IPFS hash).
+    /// @param _metadata The IPFS hash of the new metadata object.
     function setMetadata(bytes calldata _metadata) external virtual;
 
-    /// @notice Emitted when the DAO metadata is updated
-    /// @param metadata The IPFS hash of the new metadata object
+    /// @notice Emitted when the DAO metadata is updated.
+    /// @param metadata The IPFS hash of the new metadata object/
```

Comment:
I think there's an extra `/` here at the end of the sentence that should be replaced by a `.`